ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/issues/index.tsx

```diff
@@ -93,6 +93,63 @@ const defaultPageSize = 10;
 
 const columnHelper = createColumnHelper<Issue>();
 
+// Component to render a single issue card in the list.
+const IssueCard = ({ issue }: { issue: Issue }) => {
+  const packageIdType = useUserSettingsStore((state) => state.packageIdType);
+  const id =
+    packageIdType === 'ORT_ID'
+      ? identifierToString(issue.identifier)
+      : issue.purl;
+
+  return (
+    <div className='flex flex-col gap-1'>
+      <div className='flex justify-between'>
+        <div className='font-semibold'>
+          <BreakableString text={id || 'No ID available'} />
+        </div>
+        <Badge className='bg-blue-300 whitespace-nowrap' variant='small'>
+          {getIssueCategory(issue.message)}
+        </Badge>
+      </div>
+      <div className='flex justify-between'>
+        <div className='flex gap-1 text-sm'>
+          <div className='font-semibold'>Created at:</div>
+          <TimestampWithUTC
+            timestamp={issue.timestamp}
+            className='text-muted-foreground'
+          />
+        </div>
+        <div className='text-muted-foreground flex gap-1'>
+          <div>From</div>
+          <div>
+            <FormattedValue value={issue.worker} />
+          </div>
+        </div>
```

Comment:
Proposal: Make this a bit slicker by using a left-align sentence like

Created **30.09.2025** at **11:11:00** by **Scanner**

I.e. again make the actual values, not the titles, bold.

We now win the right-aligned side for some additional data, like maybe for the issue category?